.gitignore

```diff
@@ -1,5 +1,6 @@
 doc/_build
-doc/config_files.rst
+doc/using/config_files.rst
 doc/howto/user_interface.rst
 doc/howto/languages.rst
-.ipynb_checkpoints
\ No newline at end of file
+.ipynb_checkpoints
+.vscode
```

Comment:
I recommend doing something [like this](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html) so we don't end up maintaining an ignore list of everyone's favourite editors :)